gap/PGMatrix.gi

```diff
@@ -0,0 +1,431 @@
+IsPGMatrix@ := function(pgMatrix)
```

Comment:
Given that you implemented this function, I'm wondering whether it would not have been better to actually create a new object... It seems you are trying to recreate that "functionality". What do you think?